cf/app_files/app_files.go

```diff
@@ -139,9 +139,18 @@ func copyPathToPath(fromPath, toPath string) (err error) {
 
 func loadIgnoreFile(dir string) CfIgnore {
 	fileContents, err := ioutil.ReadFile(filepath.Join(dir, ".cfignore"))
+
 	if err == nil {
 		return NewCfIgnore(string(fileContents))
 	} else {
-		return NewCfIgnore("")
+		// FIXME: Should we extend to other VCS systems beyond .gitignore
+		gitContents, err2 := ioutil.ReadFile(filepath.Join(dir, ".gitignore"))
+		if err2 == nil {
+			// This works because .cfignore is same format as .gitignore
+			return NewCfIgnore(string(gitContents))
+		} else {
```

Comment:
Total stylistic nit, but since the `if err2 == nil` case returns, I'd remove the `else {}` block around the following return statement. Or:
```go
if err2 == nil {
  gitContents = ""
}
return NewCfIgnore(gitContents)
```